probe/overlay/weave.go

```diff
@@ -134,9 +134,15 @@ func (w *Weave) Tag(r report.Report) (report.Report, error) {
 	}
 
 	// Put information from weave ps on the container nodes
+	const maxPrefixSize = 12
 	for id, node := range r.Container.Nodes {
-		prefix, _ := node.Latest.Lookup(docker.ContainerID)
-		prefix = prefix[:12]
+		prefix, ok := node.Latest.Lookup(docker.ContainerID)
+		if !ok {
```

Comment:
In my repro (a Kubernetes cluster) this was failing because the lookup failed. Is this an invariant? 

I am asking in case this was just the symptom of something else being broken,